doc/source/examples/unresolved-cfd-dem/dense-pneumatic-conveying/dense-pneumatic-conveying.rst

```diff
@@ -0,0 +1,634 @@
+
+.. code-block:: text
+
+   subsection lagrangian physical properties
+     set g                        = 0, -9.81, 0
+     set number of particle types = 1
+     subsection particle type 0
+       set size distribution type            = uniform
+       set diameter                          = 0.005
+       set number of particles               = 32194
+       set density particles                 = 890
+       set young modulus particles           = 1e7
+       set poisson ratio particles           = 0.33
+       set restitution coefficient particles = 0.3
+       set friction coefficient particles    = 0.3
+       set rolling friction particles        = 0.2
+     end
+     set young modulus wall           = 1e7
+     set poisson ratio wall           = 0.33
+     set restitution coefficient wall = 0.3
+     set friction coefficient wall    = 0.4
+     set rolling friction wall        = 0.2
+   end
+
+Model Parameters
+~~~~~~~~~~~~~~~~
+Model parameters are the same as in the DEM simulation, but we do not use any strategies for performance enhancement, such as load balancing or adaptive sparse contacts.
+
+.. code-block:: text
+
+   subsection model parameters
+     subsection contact detection
+       set contact detection method = dynamic
+       set neighborhood threshold   = 1.3
+     end
+     set particle particle contact force method = hertz_mindlin_limit_overlap
+     set particle wall contact force method     = nonlinear
+     set integration method                     = velocity_verlet
+     set rolling resistance torque method       = constant_resistance
+   end
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation lasts 5 seconds and the CFD time step is 5e-4 seconds.
+
+.. code-block:: text
+
+    subsection simulation control
+      set method               = bdf1
+      set output name          = cfd_dem
+      set output frequency     = 10
+      set time end             = 5
+      set time step            = 5e-4
+      set output path          = ./output/
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The physical properties of air are the same as Lavrinec *et al*. [#lavrinec2021]_
+
+.. code-block:: text
+
+    subsection physical properties
+      subsection fluid 0
+        set kinematic viscosity = 1.5e-5
+        set density             = 1.205
+      end
+    end
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The boundary condition at the wall of the pipe is a weak function where the Dirichlet condition is weakly imposed as a no-slip condition, since we specify zero velocity. The inlet and the outlet have periodic boundaries. `See here <../../../parameters/cfd/boundary_conditions_cfd.html>`_ for more information about those boundary conditions.
+
+.. code-block:: text
+
+    subsection boundary conditions
+      set number = 2
+      subsection bc 0
+        set id   = 1
```

Comment:
Should it be 0?

Reply:
Yes, I tested a few things and I forgot to change it back, nice catch!